### PR TITLE
Removal of login.windows.net / login-us

### LINKED ISSUE #224: Replace "login.windows.net" with "login.microsoftonline.com" in the authority host list
In here: https://github.com/AzureAD/azure-activedirectory-library-for-python/blob/2a1c7ffabda0d79548533434db0cc9bbae6eefcf/adal/constants.py#L211

ADAL is still defaulting to "login.windows.net".

MSAL has removed this endpoint long time ago, and is using "login.microsoftonline.com" as a default instead: https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/6bade9faf7c832a47b13c8ac3eeba75f4084190f/msal/authority.py#L13

Can we replace "login.windows.net" with "login.microsoftonline.com" in ADAL to be consistent with MSAL as well?

### PR DESCRIPTION
**Pull request by zakramer**
Login.windows.net needs to be replaced by login.microsoftonline.com.  Also removing legacy US Gov endpoint that has been retired (login-us.microsoftonline.com)

---

Notes (by Ray): This PR will fix #224.

### CODE REVIEW

File: adal/constants.py [hunk 1]
```diff
@@ -208,12 +208,10 @@ class HttpError(object):
 
 class AADConstants(object):
 
-    WORLD_WIDE_AUTHORITY = 'login.windows.net'
+    WORLD_WIDE_AUTHORITY = 'login.microsoftonline.com'
     WELL_KNOWN_AUTHORITY_HOSTS = [
-        'login.windows.net',
```
[the remark this answers is not in the document]
Comment (zakramer) [reply adal/constants.py:213]: I would advocate that we remove it and that it should be legacy.

We can add it back but this is a legacy endpoint that needs to be phased out.

[the remark this answers is not in the document]
Comment (rayluo) [reply adal/constants.py:213]: My understanding of "phasing out" is that we no longer actively use it, and we achieve it by [this change](https://github.com/AzureAD/azure-activedirectory-library-for-python/pull/227/files#diff-b997f2b366838e521d6601e960a4b877R211).

But then, for those legacy apps (if any) who happened to already hardcode `login.windows.net` as their authority, it currently still works, until when/if that endpoint being eventually shut down. Before that, removing it from this alias list would not change anything, except causing one extra request which effectively penalize those apps with a higher latency, and penalize us with a higher service-side workload (i.e. cost).

FWIW, currently ADAL .Net also uses `login.microsoftonline.com` as default authority, but still keeps `login.windows.net` in its alias list.

**Comment by bernie-msft** [comment]
@rayluo / @abhidnya13 , what's the ETA on the test result before this PR can be merged?

**Comment by henrik-me** [comment]
ETA is this Friday

**Pull request merged**